packages/twenty-front/src/modules/object-record/record-inline-cell/components/RecordInlineCellContainer.tsx

```diff
@@ -127,18 +151,28 @@ export const RecordInlineCellContainer = ({
       onMouseLeave={handleContainerMouseLeave}
     >
       {(!!IconLabel || !!label) && (
-        <StyledLabelAndIconContainer>
-          {IconLabel && (
-            <StyledIconContainer>
-              <IconLabel stroke={theme.icon.stroke.sm} />
-            </StyledIconContainer>
-          )}
-          {showLabel && label && (
-            <StyledLabelContainer width={labelWidth}>
-              <EllipsisDisplay maxWidth={labelWidth}>{label}</EllipsisDisplay>
-            </StyledLabelContainer>
-          )}
-        </StyledLabelAndIconContainer>
+        <>
+          <StyledLabelAndIconContainer id={IconLabel?.displayName}>
+            {IconLabel && (
+              <StyledIconContainer>
+                <IconLabel stroke={theme.icon.stroke.sm} />
+              </StyledIconContainer>
+            )}
+            {showLabel && label && (
+              <StyledLabelContainer width={labelWidth}>
+                <EllipsisDisplay maxWidth={labelWidth}>{label}</EllipsisDisplay>
+              </StyledLabelContainer>
+            )}
+          </StyledLabelAndIconContainer>
+          <StyledTooltip
+            anchorSelect={`#${IconLabel?.displayName}`}
+            content={label ? label : removeWordIcon(IconLabel?.displayName)}
```

Comment:
let's use the label instead of the IconLabel
(and remove the removeWordIcon function which won't be needed anymore)

Reply:
> let's use the label instead of the IconLabel (and remove the removeWordIcon function which won't be needed anymore)

Actually the label was sometimes empty so i have used IconLabel.displayName. But now its removed